src/components/App/Header/AppHeaderMenu.vue

```diff
@@ -141,6 +141,11 @@ export default class AppHeaderMenu extends Mixins(TranslationMixin) {
 
   @mutation.settings.toggleDisclaimerDialogVisibility private toggleDisclaimerDialogVisibility!: FnWithoutArgs;
 
+  @Watch('libraryTheme', { immediate: true })
+  onLibraryThemeChange(newTheme: Theme) {
+    this.selectedTheme = newTheme === Theme.LIGHT ? HeaderMenuType.LightMode : HeaderMenuType.NoirMode;
```

Comment:
Could you create method to change selectedTheme, and apply this method in `mounted `hook too?